woocommerce/woocommerce-android
# [Simplified login] Email screen changes

Closes: #7611

### Description
This PR adds an alternative layout for the email screen to be used with the Simplified login variant.

### Testing instructions
1. Make sure `SimplifiedLoginExperiment#getCurrentVariant()` returns `LoginVariant.SIMPLIFIED`.
2. Open the app.
3. Click on "Continue with WordPress.com"
4. Confirm the email screen matches the design. 

### Images/gif
<img src="https://user-images.githubusercontent.com/1657201/197570791-522887f0-5e7c-4f9c-a8b1-24a29f0d0299.png" width=360 />
